### PR TITLE
Changed format to backticks for code and bold for tools and UI elements

### LINKED ISSUE #31: Make sure that back ticks are added to libraries, files, etc.
There are many places in the documentation, where back ticks are missing. Libraries, files and anything code related needs to be between back ticks. For example at [this link](https://staging-openms.readthedocs.io/en/staging/docs/installations/installation-on-gnu-linux.html):
Screen Shot 2022-05-23 at 8 58 35 am

### PR DESCRIPTION
**Pull request by ghost**
## Describe the change
As discussed, I've made sure backticks have been used for code and bold for tools and UI elements. Multiple files are affected.

Closes #31

## PR checklist

- [x] I have added description of the change I'm proposing in the OpenMS Documentation.
- [x] I have read and followed [OpenMS Documentation Contributing guidelines](CONTRIBUTING.md).
- [ ] I have attached a screenshot of the relevant area after this change.
- [ ] `CHANGELOG.md` is updated.
- [ ] I have added my name in [CONTRIBUTING.md](CONTRIBUTING.md#openms-documentation-contributors).

### CODE REVIEW

File: docs/installations/installation-on-gnu-linux.md [hunk 1]
```diff
@@ -12,22 +12,22 @@ sudo gdebi /PATH/TO/OpenMS.deb
 ```
 If you encounter errors with unavailable packages, troubleshoot using the following steps.
 
-1. Qt5 (or one of its packages, e.g. qt5xbase) is missing
-   It might be becuase your Debian is too old to have a recent enough version in its official repositories. It is
-   suggested to use the same packages that are used while building (make sure to adapt the Qt version and your
+1. `Qt5` (or one of its packages, e.g. `qt5xbase`) is missing.
```
Comment (tapaswenipathak) [inline docs/installations/installation-on-gnu-linux.md]: Qt5 can remain as it is.

File: docs/installations/installation-on-gnu-linux.md [hunk 2]
```diff
@@ -12,22 +12,22 @@ sudo gdebi /PATH/TO/OpenMS.deb
...
+   It might be because your Debian is too old to have a recent enough version in its official repositories. It is
+   suggested to use the same packages that are used while building (make sure to adapt the `Qt` version and your
```
Comment (tapaswenipathak) [inline docs/installations/installation-on-gnu-linux.md]: everywhere.

File: docs/installations/installation-on-gnu-linux.md [hunk 3]
```diff
@@ -12,22 +12,22 @@ sudo gdebi /PATH/TO/OpenMS.deb
...
-   You can find the missing version on [pkgs.org](https://pkgs.org) and install it with gdebi, too. You can have
+2. ICU with its `libicu` is missing.
+   You can find the missing version on [pkgs.org](https://pkgs.org) and install it with `gdebi`, too. You can have
    multiple versions of ICU installed.
 3. Error while executing a tool
-   To ensure the tool functionalry, make sure you add the `OPENMS_DATA_PATH` variable to your environmnet as follow
+   To ensure the tool functionality, make sure you add the `OPENMS_DATA_PATH` variable to your environment as follow
    `export OPENMS_DATA_PATH=/usr/share/OpenMS`
-4. Thirdparty installation of Qt5 in 1
+4. Thirdparty installation of `Qt5` in 1
```
Comment (tapaswenipathak) [inline docs/installations/installation-on-gnu-linux.md]: here as well.

File: docs/installations/installation-on-macos.md [hunk 4]
```diff
@@ -52,8 +53,8 @@ Make sure `<OpenMS-PATH>` points to the folder where OpenMS is installed locally
    cd /Applications/OpenMS-<version>
    sudo xattr -r -d com.apple.quarantine *
    ```
-2. Bug with running Java based thirdparty tools like MSGFPlusAdapter and LuciphorAdapter from within TOPPAS.app
-   If you face issues while running Java based thirdparty tools from within `TOPPAS.app`, run the TOPPAS.app from within
-   the Terminal.app (e.g. with the `open` command) to get access to the `path` where Java is located.
-   Java is usually present in the `PATH` of the terminal. Adavanced users can set this path in the `Info.plist` of/inside
-   the `TOPPAS.app`.
+2. Bug with running Java based thirdparty tools like MSGFPlusAdapter and LuciphorAdapter from within **TOPPAS.app**
+   If you face issues while running Java based thirdparty tools from within TOPPAS.app, run the TOPPAS.app from within
```
Comment (tapaswenipathak) [inline docs/installations/installation-on-macos.md:58]: can you add a newline b/t 56 and 57 line number.

File: docs/installations/installation-on-gnu-linux.md (hunk 1 above)
Comment (ghost) [reply docs/installations/installation-on-gnu-linux.md]: done

File: docs/installations/installation-on-gnu-linux.md (hunk 2 above)
Comment (ghost) [reply docs/installations/installation-on-gnu-linux.md]: done

File: docs/installations/installation-on-gnu-linux.md (hunk 3 above)
Comment (ghost) [reply docs/installations/installation-on-gnu-linux.md]: done

File: docs/installations/installation-on-macos.md (hunk 4 above)
Comment (ghost) [reply docs/installations/installation-on-macos.md:58]: done

**Review by tapaswenipathak — APPROVED** [review]

**Pull request merged**